src/Info.js

```diff
@@ -2,21 +2,29 @@ import React from 'react'
 import './Info.css'
 
 const Info = ({ user }) => {
-    return (
-        <div className='user-info-container'>
-            <div className='user-info'>
-                <h2>User Information</h2>
-                <p>Username: {user.username}</p>
-                <p>Email: {user.email}</p>
+    if (user.dogs) {
```

Comment:
This was very simple fix that seems to be working well with our invalid map error.